frontend/src/routes/MonthlyWorkingHours.jsx

```diff
@@ -217,7 +217,7 @@ const MonthlyWorkingHours = () => {
         />
       </Col>
       <Col span={19} push={5}>
-        <Table columns={columns} dataSource={tableData} />
+        <Table pagination={false} columns={columns} dataSource={tableData} />
```

Comment:
Although the order of props doesn't matter, it would be nice to have the pagination prop as last the same way it is for other tables.